gm_dotnet_managed/Tests/ConsoleTest.cs

```diff
@@ -0,0 +1,207 @@
+                    StringBuilder stringBuilderNewLine = new("☭");
+                    stringBuilderNewLine.Append(" USSR ");
+                    stringBuilderNewLine.Append('☭');
+                    Console.WriteLine(stringBuilderNewLine);
+                    Console.WriteLine((StringBuilder)null);
+                    // decimal
+                    Console.WriteLine((decimal)1234);
+                    // double
+                    Console.WriteLine((double)1234);
+                    // float
+                    Console.WriteLine((float)1234.56789);
+                    // int
+                    Console.WriteLine(1234);
+                    // uint
+                    Console.WriteLine((uint)1234);
+                    // long
+                    Console.WriteLine(12345678901);
+                    // ulong
+                    Console.WriteLine((ulong)12345678901);
+                    // object
+                    Console.WriteLine(new object());
+                    Console.WriteLine((object)null);
+                });
+                asyncTask.Wait();
+                #endregion
+                #region Async WriteLine()
+                Task asyncAwaitTask = Task.Run(async () =>
+                {
+                    // string
+                    await Console.Out.WriteLineAsync("string");
+                    await Console.Out.WriteLineAsync((string)null);
+                    // char
+                    await Console.Out.WriteLineAsync('c');
+                    // char[]
+                    await Console.Out.WriteLineAsync(new char[] { 'c', 'h', 'a', 'r', ' ', 'a', 'r', 'r', 'a', 'y' });
+                    await Console.Out.WriteLineAsync((char[])null);
+                    // char[] index length
+                    await Console.Out.WriteLineAsync(new char[] { 'c', 'h', 'a', 'r', ' ', 'a', 'r', 'r', 'a', 'y' }, 0, 10);
+                    // StringBuilder
+                    StringBuilder stringBuilderNewLine = new("☭");
+                    stringBuilderNewLine.Append(" USSR ");
+                    stringBuilderNewLine.Append('☭');
+                    await Console.Out.WriteLineAsync(stringBuilderNewLine);
+                    await Console.Out.WriteLineAsync((StringBuilder)null);
+                });
+                asyncAwaitTask.Wait();
+                #endregion
+                #region 1000 characters
+                StringBuilder limitStringBuilder = new();
+                Random random = new(0);
+                for (uint i = 0; i < 1000; i++)
+                {
+                    limitStringBuilder.Append(random.Next(0, 10));
+                }
+                Console.WriteLine(limitStringBuilder);
+                #endregion
+
+                taskCompletion.TrySetResult(true);
+            }
+            catch (Exception e)
+            {
+                taskCompletion.TrySetException(e);
+            }
+
+            return taskCompletion.Task;
+        }
+    }
+}
```

Comment:
This test is not really automatic: one should look to the output and check if it is correct. I guess we must remove whole test here. Latter I will add special unit test for it by creating a native library which will act like Tier0.

Reply:
resolved by https://github.com/GmodNET/GmodDotNet/pull/70/commits/6d7b547ee49ec3e242082b3451f917b83e72ea6f